### ISSUE TITLE
Manifest HEAD endpoint

### ISSUE THREAD

**Issue by olegmoz** [artipie/docker-adapter#119]
By specification Manifest entity is expected to respond to `HEAD` method requests in the same manner as `GET` request, but without body. See https://docs.docker.com/registry/spec/api/#detail

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
The job [#119](https://github.com/artipie/docker-adapter/issues/119) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/e7a9a0db-28dc-44d2-ad0a-d555d19d593f); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by olenagerasimova** [comment]
@olegmoz could you please explain why current implementation of `com.artipie.docker.http.ManifestEntity.Get` does not add `Docker-Content-Digest` header as specified in https://docs.docker.com/registry/spec/api/#manifest?

**Comment by olegmoz** [comment]
@olenagerasimova There is no any reason for that and it may be considered to be a bug

**Comment by olenagerasimova** [comment]
@olegmoz thanks for response, I've added todo to implement `Docker-Content-Digest` in manifest endpoints

**Comment by 0pdd** [comment]
@olegmoz 2 puzzles [#128](https://github.com/artipie/docker-adapter/issues/128), [#129](https://github.com/artipie/docker-adapter/issues/129) are still not solved.

**Comment by 0crat** [comment]
Job was finished in 45 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@771b26b8-67e1-41d2-9b59-49db76911c64, thread: PQ-CT2E6TK9B
com.zerocracy.farm.ruled.RdRules[130] java.lang.IllegalStateException: http://datum.zerocracy.com/latest/rules/pm/no-lost-boosts.xsl says: The boost for job "gh:artipie/docker-adapter#119" exists in boosts.xml, but the job is absent in the WBS among 77 jobs.
```

[0.53.14](https://github.com/zerocracy/farm): CID: [64055e12-42eb-4cc1-9c58-da7545fa7359](https://www.0crat.com/footprint/CT2E6TK9B/64055e12-42eb-4cc1-9c58-da7545fa7359), Type: "Remove job from WBS"

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0pdd** [comment]
@olegmoz the puzzle [#129](https://github.com/artipie/docker-adapter/issues/129) is still not solved; solved: [#128](https://github.com/artipie/docker-adapter/issues/128).

**Comment by 0pdd** [comment]
@olegmoz all 2 puzzles are solved here: [#128](https://github.com/artipie/docker-adapter/issues/128), [#129](https://github.com/artipie/docker-adapter/issues/129).
